apps/etterlatte-vedtaksvurdering/src/main/kotlin/vedtaksvurdering/VedtakBehandlingService.kt

```diff
@@ -536,6 +522,29 @@ class VedtakBehandlingService(
         }
     }
 
+    @TestOnly
```

Comment:
Denne metoden blir jo vel no kalt frå `fattVedtak`-metoden, så da blir denne annotasjonen misvisande.

Reply:
Har egentlig brukt den som erstatning for gode gamle `@VisibleForTesting`, men kan hende det er mer hensiktsmessig å flytte funksjonen til en helperklasse e.l. i stedet.

Reply:
Flyttet!